src/atc/transformers/union_transformer.py

```diff
@@ -0,0 +1,18 @@
+from pyspark.sql import DataFrame
+
+from atc.etl import Transformer
+from atc.etl.types import dataset_group
+
+
+class UnionTransformer(Transformer):
+    """Returns the union of all input dataframes."""
+
+    def process(self, df: DataFrame) -> DataFrame:
+        return df
+
+    def process_many(self, datasets: dataset_group) -> DataFrame:
+        dfs = list(datasets.values())
+        df = dfs[0]
+        for other in dfs[1:]:
+            df = df.union(other)
```

Comment:
Please keep in mind the difference between union and unionByName. https://spark.apache.org/docs/3.1.1/api/python/reference/api/pyspark.sql.DataFrame.unionByName.html

Reply:
also remember that if the dataframes has different columns, you can set allowMissingColumns=True. Again, see documentation.